cmake/modules/FindKOKKOS.cmake

```diff
@@ -44,10 +44,20 @@ ELSE()
       STRING(REGEX REPLACE "\\$<\\$<LINK_LANGUAGE:CXX>:([^>]*)>" "\\1" KOKKOS_EXTRA_LD_FLAGS "${KOKKOS_EXTRA_LD_FLAGS_FULL}")
       GET_PROPERTY(KOKKOS_COMPILE_FLAGS_FULL TARGET Kokkos::kokkoscore PROPERTY INTERFACE_COMPILE_OPTIONS)
       STRING(REGEX REPLACE "\\$<\\$<COMPILE_LANGUAGE:CXX>:([^>]*)>" "\\1" KOKKOS_COMPILE_FLAGS "${KOKKOS_COMPILE_FLAGS_FULL}")
-      # In serial the flag is empty but ADD_FLAGS does not support adding an empty
-      # flag
-      IF(KOKKOS_COMPILE_FLAGS)
-        ADD_FLAGS(DEAL_II_CXX_FLAGS ${KOKKOS_COMPILE_FLAGS})
+      STRING(REGEX REPLACE ";" " " KOKKOS_COMPILE_FLAGS_STR "${KOKKOS_COMPILE_FLAGS}")
```

Comment:
```suggestion
      STRING(REGEX REPLACE ";" " " KOKKOS_CXX_FLAGS "${KOKKOS_COMPILE_FLAGS}")
```

Reply:
You just need to set the `KOKKOS_CXX_FLAGS` variable and the build system should pick up the flags automatically (and properly report where they are coming from in `detailed.log`). :slightly_smiling_face: